### PR TITLE
Bring aaa.asp into existence 

### PR DESCRIPTION
**Pull request by dispherical**

### CODE REVIEW

**Comment by Benricheson101** [comment]
is it possible to make it add one `a` at a certain interval? similarly to how aaa.html works https://github.com/Benricheson101/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa/blob/af2548f0cfade9eae04e50cd0a3bb8e1cb3f16f2/aaa.html#L5

**Comment by dispherical** [comment]
I'm not really sure. I barely write anything in ASP

I can look it up

**Comment by trentwiles** [comment]
Probably not because ASP is server side (Active Server Pages)

**Comment by dispherical** [comment]
Yeah, I could not find anything

**Comment by dispherical** [comment]
Should I do this in Javascript also?

File: aaa.asp [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<head>
+<title><%
+response.write("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
```
Comment (democat3457) [inline aaa.asp:4]: It'd probably be unwise to keep adding on to the title, might make the supposed browser crash?

File: aaa.asp [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+%></title>
+</head>
+<h1><%
+response.write("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
+%></h1><p><%
```
Comment (democat3457) [inline aaa.asp:9]:
```suggestion
<p><%
```
It would be adding two per iteration and on separate lines rather than just one

File: aaa.asp [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+response.write("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
```
Comment (democat3457) [inline aaa.asp:10]:
```suggestion
Do Until False
  response.write("a")
Loop
```

This *should* work, I'm not sure if it'll print on new lines?

**Review by democat3457 — CHANGES_REQUESTED** [review]
asp has such a limited documentation... but this cleans up the code a bit and actually prints 'a' now

**Pull request closed without merging**